swagger_server/controllers_local/controller_helper.py

```diff
@@ -0,0 +1,18 @@
+from uuid import uuid4
+from structlog import get_logger
+
+logger = get_logger()
+
+
+def ensure_log_on_error(code, msg):
```

Comment:
This needs to be expanded, any error being recorded as info will be lost if we ever needed to debug